test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -586,6 +586,27 @@ def forward(self, x):
         x = torch.arange(-5, 5).to(dtype=torch.float32)
         self.run_test(MyModel(), x)
 
+    def test_hardswish(self):
```

Comment:
Can we add this op in the list of supported ops on the doc page: https://pytorch.org/docs/master/onnx.html?